examples/separate-config/every-feature-multiple-dest-single-ms.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
```

Comment:
This shows every feature with a single multiselect instance having multiple destinations.
I use a single UNDO/REDO button combo for this. Some IDs may not be semantically correct yet.